index.html

```diff
@@ -4,64 +4,68 @@
     <meta charset="UTF-8">
     <title>Fork Me! FCC: Test Suite Template</title>
     <link rel="stylesheet" href="css/style.css">
+     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+      <script  src="js/index.js"></script>
   </head>
 
   <body>
-    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-     <nav id ="navbar" >
-       <header>this documentation is a sum of all of mankind knowledge</header>
+    <header>
+     <nav id="navbar">
+       <h2>this documentation is a sum of all of mankind knowledge</h2>
```

Comment:
- Please use inline comments like this one in your lines of code that need to change not change them you need to make a review not fix bugs.
- Kindly make the comments polite and explain the reason for each comment.